src/redux/stores/interests.js

```diff
@@ -0,0 +1,14 @@
+const interests = {
+  Sports: false,
```

Comment:
I think we need to understand which data we should to submit on backend before creating store for interests

Reply:
done